### ISSUE TITLE
Try model.eval() after loading the model

### ISSUE THREAD

**Issue by martinbaerwolff** [OpenTrafficCam/OTVision#132]
Remember that you must call to set dropout and batch normalization layers to evaluation mode before running inference. Failing to do this will yield inconsistent inference results.
Source: https://pytorch.org/tutorials/beginner/saving_loading_models.html#save-load-entire-model
